src/map/npc.cpp

```diff
@@ -352,6 +354,26 @@ uint64 StylistDatabase::parseBodyNode( const ryml::NodeRef& node ){
 			entry->value = value;
 		}
 
+#if PACKETVER >= 20231220
+		if (constant == LOOK_BODY2 && this->nodeExists(node, "RequiredJob")) {
```

Comment:
```suggestion
		if (this->nodeExists(node, "RequiredJob")) {
```
I would not limit this to LOOK_BODY2, actually could be used for the rest as well.